package.json

```diff
@@ -32,8 +32,43 @@
   },
```

Comment:
Should the `gitignore` be modified to allow for checking in the `package-lock.json` file?